recipes/sixtools/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "sixtools" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 38950e920b9ae28979f1dee66756763714b601ed1553ec864a8278de6c295105
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Making the install comamnd consitent:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```